Add isEmpty documentation file for Swift Arrays (#2655)
Codecademy/docs: content/swift/concepts/arrays/terms/isEmpty/isEmpty.md

---
Title: '.isEmpty'
Description: 'Returns true if an array contains no elements, returns false if the array contains one or more elements.'
Subjects:
  - 'iOS'
  - 'Mobile Development'
Tags:
  - 'Arrays'
  - 'Methods'
CatalogContent:
  - 'learn-swift'
  - 'paths/build-ios-apps-with-swiftui'
---

The **`.isEmpty`** method returns true if an array contains no elements, returns false if the array contains one or more elements.

## Syntax

```pseudo
arrayName.isEmpty
```

## Example

```swift
var favoriteSongs = [String]()

print(favoriteSongs.isEmpty)
//Output: true

favoriteSongs.append("Stayin' Alive")
print(favoriteSongs.isEmpty)

> //Output: false

Reviewer: why don't you create a output section for the outputs. Something like this
<img width="457" alt="image" src="https://github.com/Codecademy/docs/assets/94860622/0ca0732e-3efa-47d8-86f3-04e4b51e9c05">

Author: I will make this change. Thank you for the feedback!

---
Title: '.isEmpty'
Description: 'Returns true if an array contains no elements, returns false if the array contains one or more elements.'
Subjects:
  - 'iOS'
  - 'Mobile Development'
Tags:
  - 'Arrays'
  - 'Methods'
CatalogContent:
  - 'learn-swift'
  - 'paths/build-ios-apps-with-swiftui'
---

> The **`.isEmpty`** method returns true if an array contains no elements, returns false if the array contains one or more elements.

Reviewer 2: ```suggestion
The **`.isEmpty`** property returns true if an array contains no elements, returns false if the array contains one or more elements.
```

## Syntax

```pseudo
arrayName.isEmpty

> ```

Reviewer 2: There should be a sentence after the syntax block explaining what `arrayName` is.

## Example

> 

Reviewer 2: There should be a sentence before the example summarizing what it is doing.

---
Title: '.isEmpty'
Description: 'Returns true if an array contains no elements, returns false if the array contains one or more elements.'
Subjects:
  - 'iOS'
  - 'Mobile Development'
Tags:
  - 'Arrays'
  - 'Methods'
CatalogContent:
  - 'learn-swift'
  - 'paths/build-ios-apps-with-swiftui'
---

> The **`.isEmpty`** property returns true if an array contains no elements, returns false if the array contains one or more elements.

Reviewer 3: ```suggestion
The **`.isEmpty`** property returns `true` if an array contains no elements, and returns `false` if the array contains one or more elements.
```

## Syntax

```pseudo
arrayName.isEmpty
```

> Where `arrayName` is the arry being checked for elements.

Reviewer 3: ```suggestion
Where `arrayName` is the array being checked for elements.
```

## Example

> The following example creates and array and checks `.isEmpty()` before and after adding an element:

Reviewer 3: ```suggestion
The following example creates and array and evaluates the contents of the array with `.isEmpty` before and after adding an element:
```
